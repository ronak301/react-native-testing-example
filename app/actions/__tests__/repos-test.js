import { selectRepo } from '../repos';

it('creates a SELECT_REPO action', () => {
  expect(selectRepo(1000)).toMatchSnapshot();
});