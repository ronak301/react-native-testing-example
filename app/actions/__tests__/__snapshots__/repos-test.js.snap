exports[`test creates a SELECT_REPO action 1`] = `
Object {
  "id": 1000,
  "type": "SELECT_REPO"
}
`;
